docs/validation-server/DESIGN.md

```diff
@@ -0,0 +1,27 @@
+The Plutus Core validation server will behave as follows:
+
+  - Bind to port 3000
+
+  - Accept HTTP requests with a JSON body
+
+  - The server should accept `GET` requests that contain the JSON data for
+    validation in the request body
+
+    - As an example:
+      ```json
+      {
+          "validator": "ab34ce2d",
+          "redeemer": "5cb418",
+          "data": "958bce"
+      }
```

Comment:
Missing closing backticks.

You also need to provide the information about the pending transaction.